core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -620,8 +620,9 @@ public static boolean checkIfPrefixExists(String path) {
     }
 
     final String lowerPath = path.toLowerCase(Locale.getDefault());
-    return lowerPath.contains("://") || lowerPath.startsWith(CarbonCommonConstants.HDFSURL_PREFIX)
-        || lowerPath.startsWith(CarbonCommonConstants.VIEWFSURL_PREFIX) || lowerPath
+    return lowerPath.contains(":/") || lowerPath.contains("://") || lowerPath
```

Comment:
why `lowerPath.contains(":/")` is added ? It is for which file system ?

Reply:
removed

Reply:
@ajantha-bhat On local while running test cases the path starts with file:/ instead of file://, that is why i have added this back.

Reply:
ok. 
I think this is happening with newly added FS scheme. 
Because without scheme these test cases were passing.  May be we would have checked scheme adding method.